### PR TITLE
Use snprintf in tests

### PR DESCRIPTION
**Pull request by bobpaw**
- I missed some sprintf calls before. I don't see any others (find + grep).
- These deprecation errors come up with LLVM so others might not notice.

### CODE REVIEW

File: test/viz.cc [hunk 1]
```diff
@@ -65,7 +65,7 @@ int main(int argc, char** argv)
 
     }
     v.markPart(m,part_num);
-    sprintf(output,"Testing MIS %d",i);
+    snprintf(output,256,"Testing MIS %d",i);
```
Comment (cwsmith) [inline test/viz.cc:68]: Should this be 128?

**Review by cwsmith — CHANGES_REQUESTED** [review]
Thank you for the changes.  I may have found one size mismatch.

File: test/viz.cc (hunk 1 above)
Comment (bobpaw) [reply test/viz.cc]: Yep. Fixed it.

**Review by cwsmith — APPROVED** [review]

**Pull request merged**